populus/config/base.py

```diff
@@ -53,8 +53,14 @@ def get_master_config(self):
         else:
             return self.parent.get_master_config()
 
+    def has_references(self):
+        if [item for item in self.items(flatten=True) if item[0].rpartition(".")[2]=="$ref"]:  # noqa
```

Comment:
I'd like to avoid blanket `# noqa` statements because they can hide other errors.  Can you change this to `# noqa: E501` to silence the specific error code?